src/javaRosa.js

```diff
@@ -951,20 +948,18 @@ define([
                 widget.setValue(defaultValue);
                 widget.handleChange();
             } else {
-                var itextItem = widget.getItextItem();
+                var value = widget.getItextValue();
 
-                if (!itextItem) {
-                    widget.setValue("");
-                    return;
+                if (!_.isString(value)) {
```

Comment:
It only catches `undefined`. I originally used `!_.isUndefined` but like that this makes it obvious we only expect strings